tools/linters/go.mod

```diff
@@ -4,9 +4,11 @@ go 1.17
 
 require (
 	github.com/golangci/golangci-lint v1.39.0
+	go.opentelemetry.io/build-tools/dbotconf v0.0.0-20220304232821-55b850922377
 	go.opentelemetry.io/build-tools/multimod v0.0.0-20210920164323-2ceabab23375
```

Comment:
I'd suggest updating multimod to the same pseudoversion.  I don't think anything in their common dependencies will have broken multimod at this version if updated to the dbotconf version, but it's a possibility if they diverge.

Reply:
Thank you, that's a good point. Updated.